src/Stripe.net.Tests/Stripe.net.Tests.csproj

```diff
@@ -49,4 +49,8 @@
     <Reference Include="System" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```

Comment:
Same with this one.

Reply:
Good point. Those were added during the build. However, this isn't the case anymore and it seems they can be safely removed. I'll amend the commit.

Reply:
These are added by VS to indicate the given project is a test project and that the test runner should be loaded for it.